src/button.ts

```diff
@@ -1,4 +1,4 @@
-import { InlineKeyboardButton, KeyboardButton } from './telegram-types'
+import { InlineKeyboardButton, KeyboardButton } from 'typegram'
```

Comment:
I think this should not be a direct dependency to typegram, but use the new module instead